Sources/ViewTypes/TextFieldWithVerticalAxis.swift

```diff
@@ -96,11 +97,12 @@ extension tvOSViewVersion<TextFieldWithVerticalAxisType, UITextField> {
     public static let v15 = Self.unavailable()
 
     public static let v16 = Self(for: .v16)
-    public static let v17 = Self(for: .v17)
```

Comment:
Is this expected?

Reply:
Oops, that was a mistake, thanks for catching it!